package.json

```diff
@@ -8,11 +8,28 @@
   },
   "license": "MIT",
   "homepage": "https://github.com/code-dot-org/dsco_#readme",
-  "scripts": {},
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
+    "build-storybook": "build-storybook",
+    "storybook": "start-storybook -p 6006"
+  },
   "devDependencies": {
+    "@babel/core": "^7.16.7",
     "@commitlint/cli": "^16.0.2",
     "@commitlint/config-conventional": "^16.0.0",
+    "@storybook/addon-actions": "^6.4.12",
+    "@storybook/addon-essentials": "^6.4.12",
+    "@storybook/addon-links": "^6.4.12",
+    "@storybook/builder-webpack5": "^6.4.12",
+    "@storybook/manager-webpack5": "^6.4.12",
+    "@storybook/react": "^6.4.12",
+    "babel-loader": "^8.2.3",
     "husky": "^7.0.4",
-    "lerna": "^4.0.0"
-  }
+    "lerna": "^4.0.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
added these because storybook requires that react and react-dom be in the root scope